internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -160,8 +168,13 @@ func (*Sequencer) Boot(r runtime.Runtime) []runtime.Phase {
 
 	phases = phases.AppendWhen(
 		r.State().Platform().Mode() != runtime.ModeContainer,
-		"mountBoot",
+		"mountSystem",
 		MountBootPartition,
+		MountSystemPartition,
+	).AppendWhen(
+		r.State().Platform().Mode() != runtime.ModeContainer,
+		"mountEFI",
```

Comment:
can we skip that? as only installer touches this ever, and installer blows up partition table anyways?

Reply:
Yep, I think so.